enclave-runtime/src/tls_ra/mod.rs

```diff
@@ -57,6 +60,7 @@ impl From<u8> for Opcode {
 			0 => Opcode::ShieldingKey,
 			1 => Opcode::StateKey,
 			2 => Opcode::State,
+			3 => Opcode::LightClient,
 			_ => unimplemented!("Unsupported/unknown Opcode for MU-RA exchange"),
```

Comment:
We only use this code on the tls-client side during initialization, still this is a runtime panic that could occur. Not nice: https://github.com/integritee-network/worker/issues/1400